templates/swagger/v1_json.tmpl

```diff
@@ -17165,7 +17235,7 @@
     "parameterBodies": {
       "description": "parameterBodies",
       "schema": {
-        "$ref": "#/definitions/PullReviewRequestOptions"
+        "$ref": "#/definitions/CreateTagOption"
```

Comment:
Is this wrong before?

Reply:
@lunny no this is a hack we use to generate swagger doc ...

Reply:
so it's alright

Reply:
@lunny do you need more info on that?